path_config.go

```diff
@@ -44,6 +52,21 @@ func (b *backend) pathConfig() *framework.Path {
 				Description:      "The password for PCF’s API.",
 				DisplaySensitive: true,
 			},
+			"login_max_minutes_old": {
```

Comment:
For consistency, consider making these our normal framework.TypeDurationSeconds, which will let users express the times as they currently do for TTLs and other durations. If that's done, then changing the field names to *_duration will allow them to be printed nicely on the CLI.

Reply:
Ah! Good idea. Yes, greater usability and greater granularity. I will update them.